pandas/core/ops.py

```diff
@@ -721,6 +686,10 @@ def safe_na_op(lvalues, rvalues):
                 return na_op(lvalues, rvalues)
         except Exception:
             if isinstance(rvalues, ABCSeries):
+                # TODO: This case is not hit in tests.  For it to be hit would
+                # require `right` to be an object such that right.values
```

Comment:
limit the comments to useful info, 'screwup' does not fall in this category

Reply:
This comment is more a suggestion that we get rid of this block.

Reply:
I get that. but this is public code, you can certainly say 'this should be removed'

Reply:
ok with removing this code here